buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ThirdPartyAuditTask.java

```diff
@@ -63,17 +64,20 @@
         "\\s\\sin ([a-zA-Z0-9$.]+) \\(.*\\)"
     );
 
-    /**
-     * patterns for classes to exclude, because we understand their issues
-     */
-    private Set<String> excludes = new TreeSet<>();
+    private Set<String> missingClassExcludes = new TreeSet<>();
```

Comment:
Why TreeSets? the order should not matter?

Reply:
I'm not sure I understand. We don't print this set; we only use it to check a class that was caught missing by running forbidden apis right?

Reply:
We do print it if it contains unnecessary entries and you could have allot of them say after a dependency upgrade.